srcpkgs/helio-sequencer/template

```diff
@@ -0,0 +1,48 @@
+# Template file for 'helio-sequencer'
+pkgname=helio-sequencer
+version=3.12
+revision=1
+_gitrev="61f7db380f083b424418d65f4e26addc1370ddea"
+archs="i686 x86_64"
+hostmakedepends="git pkg-config"
+makedepends="freetype-devel libX11-devel libXinerama-devel libXrandr-devel libXcursor-devel libXcomposite-devel MesaLib-devel libfreeglut-devel libcurl-devel alsa-lib-devel jack-devel libsigc++-devel"
+depends="curl freetype libXinerama libglvnd alsa-lib"
+short_desc="One music sequencer for all major platforms, desktop and mobile"
+maintainer="IFoundSilentHouse <adeptslab@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://helio.fm/"
+changelog="https://github.com/helio-fm/helio-sequencer/blob/develop/CHANGELOG.md"
+
+do_fetch() {
+	# commit 61f7db3 is a commit of stable version 3.12
+	# helio-sequencer has thirdparty repos in its' github 
+	# so we have to use this do_fetch() hack
+	git clone --recursive https://github.com/helio-fm/helio-sequencer.git $wrksrc
+	cd $wrksrc
+	git reset --hard ${_gitrev}
+}
+
+do_build() {
+	case "$XBPS_TARGET_MACHINE" in
+		i686) export CONFIG=Release32 ;;
+		x86_64) export CONFIG=Release64 ;;
+	esac
+	cd $wrksrc/Projects/LinuxMakefile/
+	make all strip
+}
+
+do_install() {
+	cd $wrksrc
+	vbin Projects/LinuxMakefile/build/helio
+}
+
+post_install() {
+	cd $wrksrc
+	vinstall Projects/Deployment/Linux/Debian/x64/usr/share/applications/Helio.desktop 644 usr/share/applications
+	cd $wrksrc/Projects/Deployment/Linux/Debian/x64/usr/share/icons/hicolor/
```

Comment:
Don't change directories like this, unless you're running a script that really expects to be run from a particular directory. Just specify paths relative to `$wrksrc`.

Reply:
Ok. I just wanted to add readability but if you say so...

Reply:
If you want to improve readability, define something like `assets=Projects/Deployment/Linux/Debian/x64` and use that variable.